week10/news/article.go

```diff
@@ -0,0 +1,51 @@
+package news
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+type Article struct {
+	id			int
+	category 	string
+	content 	string
+	title		string
```

Comment:
Consider making most, if not all of these fields exported. You can then remove the need for the methods and the custom JSON marshaling. As long as the subscriber is given a value type, and not a pointer, they won't be able to make changes to the article outside of their own scope, so there is little point in hiding this information.